runner/mysqld_exporter.go

```diff
@@ -0,0 +1,97 @@
+// pmm-agent
+// Copyright (C) 2018 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package runner
+
+import (
+	"context"
+	"fmt"
+	"github.com/sirupsen/logrus"
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/percona/pmm-agent/utils/logger"
+)
+
+type MySQLdExporter struct {
```

Comment:
We will not need separate types for separate agents. All parameters (flags, env variables, etc.) will be passed by pmm-managed.